testdata/p4_16_samples_outputs/pna-dpdk-bvec_union-midend.p4

```diff
@@ -0,0 +1,105 @@
+#include <core.p4>
+#include <pna.p4>
+
+struct alt_t {
+    bit<1> valid;
+    bit<7> port;
+}
+
+struct row_t {
+    alt_t alt0;
+    alt_t alt1;
+}
+
+header Hdr1 {
+    bit<8> _a0;
+    bit<1> _row0_alt0_valid1;
+    bit<7> _row0_alt0_port2;
+    bit<1> _row0_alt1_valid3;
+    bit<7> _row0_alt1_port4;
+    bit<1> _row1_alt0_valid5;
+    bit<7> _row1_alt0_port6;
+    bit<1> _row1_alt1_valid7;
+    bit<7> _row1_alt1_port8;
+}
+
+header Hdr2 {
+    bit<16> _b0;
+    bit<1>  _row_alt0_valid1;
+    bit<7>  _row_alt0_port2;
+    bit<1>  _row_alt1_valid3;
+    bit<7>  _row_alt1_port4;
+}
+
+header_union U {
```

Comment:
shouldn't this be gone?

Reply:
This output is coming from p4test backend. Currently, this pass is only invoked from DPDK midend. I quickly tried enabling this in p4test midend and see many regressions. I will fix those and update the PR.